src/main/java/org/folio/gobi/RetrieveMappingDetailsService.java

```diff
@@ -0,0 +1,48 @@
+package org.folio.gobi;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.rest.mappings.model.OrderMappings;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class RetrieveMappingDetailsService {
```

Comment:
This class name ought to not be a verb, more like a noun. `MappingDetailsService` might suffice.